internal/store/postgres/scheduler/job_operator_repository.go

```diff
@@ -0,0 +1,107 @@
+package scheduler
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+
+	"github.com/odpf/optimus/core/scheduler"
+	"github.com/odpf/optimus/internal/errors"
+)
+
+const (
+	sensorRunTableName = "sensor_run"
+	taskRunTableName   = "task_run"
+	hookRunTableName   = "hook_run"
+)
+
+type OperatorRunRepository struct {
+	db *gorm.DB
+}
+
+type operatorRun struct {
+	ID       uuid.UUID `gorm:"primary_key;type:uuid;default:uuid_generate_v4()"`
+	JobRunID uuid.UUID
+
+	Name         string
+	OperatorType string
+	State        string
+
+	StartTime time.Time `gorm:"not null"`
+	EndTime   time.Time `gorm:"default:TIMESTAMP '3000-01-01 00:00:00'"`
+
```

Comment:
in my opinion we should also , keep a remarks column , to note the failure msg in case the run fails. 

@sbchaos @sravankorumilli  what are your thoughts ?

Reply:
add a TODO and fix later